pylint/checkers/classes.py

```diff
@@ -2299,6 +2303,17 @@ def _is_iterator(node):
                     pass
         return False
 
+    @staticmethod
+    def _function_body_is_ellipsis(node: nodes.FunctionDef) -> bool:
+        if (
+            len(node.body) == 1
+            and isinstance(node.body[0], nodes.Expr)
+            and isinstance(node.body[0].value, nodes.Const)
+            and node.body[0].value.value == Ellipsis
+        ):
+            return True
+        return False
```

Comment:
```suggestion
        return (
            len(node.body) == 1
            and isinstance(node.body[0], nodes.Expr)
            and isinstance(node.body[0].value, nodes.Const)
            and node.body[0].value.value == Ellipsis
        )
```
No need to add an if statement.